### DISCUSSION TITLE
Autologin

### DISCUSSION THREAD

**Discussion by Ghosttizzi** [prasanthrangan/hyprdots#713] [Q&A]
can someone guide me how to get autologin on hyprland

**Comment by kRHYME7** [comment]
Hello what specifics do you need to auto login?  Do you want to use sddm or any display manager? or just the tty auto login?
I'm  using the latter one.

**Reply by Ghosttizzi** [reply]
IDK I would like to skip the login screen. In the simplest way possible

**Reply by kRHYME7** [reply]
https://wiki.archlinux.org/title/SDDM#Autologin
I assumed you have this already installed, as this is pre-installed for Hyprdots. This is the simplest way to do it. 

Another method is modifying Getty.  (Not straightforward but lightweight and faster, it might cause your user to not even open)
Here are the links to how I successfully implemented this on my system.
https://wiki.archlinux.org/title/getty#Automatic_login_to_virtual_console
https://www.reddit.com/r/hyprland/comments/127m3ef/starting_hyprland_directy_from_systemd_a_guide_to/
https://unix.stackexchange.com/questions/42359/how-can-i-autologin-to-desktop-with-systemd

**Reply by kRHYME7** [reply]
Sorry for that here's the simplest way #715 
https://github.com/prasanthrangan/hyprdots/issues/715#issuecomment-1880814648
